scripts/audit-consumers.ts

```diff
@@ -0,0 +1,201 @@
+import zlib from "node:zlib";
+import stream from "node:stream/promises";
+
+import * as cheerio from "cheerio";
+import { Octokit } from "@octokit/rest";
+
+import { features as webFeatures } from "../index.js";
+
+// Yield all entries from chromestatus.com. Because the newest entry is returned
+// first and entries might be created while we're iterating, it's possible that
+// the same entry is yielded multiple times. If this is a problem they have to be
+// deduplicated by ID.
+async function* chromeStatusFeatures() {
+  const num = 500;
+  for (let start = 0 /* nothing here */; ; start += num) {
+    const url = `https://chromestatus.com/api/v0/features?start=${start}&num=${num}`;
+    const resp = await fetch(url);
+    if (!resp.ok) {
+      throw new Error(`Bad API response: ${resp.statusText}`);
+    }
+    const text = await resp.text();
+    if (!text.startsWith(")]}'\n")) {
+      throw new Error(
+        "Expected API response did not begin with the magic sequence.",
+      );
+    }
+    const data = JSON.parse(text.substring(5));
+    // Iteration is done when the API returns no features. This does mean that
+    // we make one more request than necessary, but it's simple and works.
+    if (!data.features.length) {
+      break;
+    }
+    for (const f of data.features) {
+      yield f;
+    }
+  }
+}
+
+async function chromiumSourceFile(path) {
+  const url = `https://github.com/chromium/chromium/raw/refs/heads/main/${path}`;
+  const resp = await fetch(url);
+  if (!resp.ok) {
+    throw new Error(`Bad response: ${resp.statusText}`);
+  }
+  return await resp.text();
+}
+
+async function chromiumUseCounters() {
+  const text = await chromiumSourceFile(
+    "tools/metrics/histograms/metadata/blink/enums.xml",
+  );
+  const $ = cheerio.load(text);
```

Comment:
This works but I don't love it. It doesn't actually parse as XML it seems, the resulting tree has all of the `<int>` elements nested in another and it only works because I made the next line work regardless of tree shape.

Parsing webdx_feature.mojom directly is an option, but any difference in parsing from the Python code that parses it in Chromium would be a risk.